docs/developer/CONTRIBUTING.md

```diff
@@ -0,0 +1,61 @@
+# Contributing
+OSIDB is an open initiative and we welcome any help.
+Before contributing please read and follow these guidelines.
+
+## How to contribute to OSIDB
+
+### Did you find a bug?
+
+* Ensure the bug was not already reported by searching on GitHub under
+  [Issues](https://github.com/RedHatProductSecurity/osidb/issues).
+
+* If you're unable to find an open issue addressing the problem:
+
+  * If you are [Red Hat](https://www.redhat.com/) associate and you are requesting company specific
+    functionality open a new issue
+    [here](https://issues.redhat.com/secure/CreateIssueDetails!init.jspa?pid=12332734&issuetype=1).
+
+  * Otherwise open a new issue [here](https://github.com/RedHatProductSecurity/osidb/issues/new).
+
+* Be sure to include a title and clear description, as much relevant information as possible,
+  and a comand line or code sample or an executable test case demonstrating the
+  expected behavior that is not occurring.
+
+### Did you write a patch that fixes a bug?
+
+* Open a new GitHub pull request with the patch.
+
+* Ensure the PR description clearly describes the problem and solution.
+  Include the relevant issue number if applicable.
+
+* Before submitting make sure that or linters and tests are passing.
```

Comment:
`that or linters and tests` -> `that linters and tests` ?

Reply:
Good catch!